pcf/cli/commands/terminate.py

```diff
@@ -0,0 +1,45 @@
+""" Logic for pcf stop command """
+
+import click
+from pcf.core import State
+from pcf.cli.utils import execute_applying_command
+
+
+@click.command(
+    name="terminate", short_help="Set desired state to 'terminated' and apply changes"
+)
+@click.option(
+    "-f",
+    "--file",
+    "file_",
+    type=click.Path(dir_okay=False, resolve_path=True),
+    default="pcf.json",
+    show_default=True,
+    help="The JSON or YAML file defining your infrastructure configuration",
+)
+@click.option(
```

Comment:
anyway to share these options so you dont have to rewrite them every-time
maybe using decorators?

https://stackoverflow.com/questions/40182157/python-click-shared-options-and-flags-between-commands

Reply:
Yeah that's a good call.